cpp-client/deephaven/client/flatbuf/deephaven/flatbuf/Barrage_generated.h

```diff
@@ -162,8 +162,8 @@ struct BarrageMessageWrapper FLATBUFFERS_FINAL_CLASS : private flatbuffers::Tabl
   }
   bool Verify(flatbuffers::Verifier &verifier) const {
     return VerifyTableStart(verifier) &&
-           VerifyField<uint32_t>(verifier, VT_MAGIC) &&
```

Comment:
(Maybe we discussed this and I forgot, but)
It looks to me like this file is the output of running flatc on barrage at or after the barrage commit 8dc0ce9e1636aab87e14060be640124d7d37e597, but before the barrage commit aab6d4aeab02bd97da9ad4abe869b1b929ca745b

(and both of those are newer than the barrage v0.4.0 tag)

Was there a rationale for why we ran flatc at that specific point? Is this a release tagged in the repository somewhere? (I don't see any tags for anything newer than 0.4.0)

Reply:
This should have been at v0.4.1 tag which is what Nate told me was the version used by the server (and I think I confirmed from server gradle code).

Ok confirmed: v0.4.1:

```
cfs@erke 22:24:58 ~/dh/oss2/barrage
$ git checkout v0.4.1
Note: switching to 'v0.4.1'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at a87771a [maven-release-plugin] prepare release v0.4.1

cfs@erke 22:25:04 ~/dh/oss2/barrage
$ cd format/

cfs@erke 22:25:09 ~/dh/oss2/barrage/format
$ ~/dhcpp/local/flatbuffers/bin/flatc --cpp Barrage.fbs

cfs@erke 22:25:29 ~/dh/oss2/barrage/format
$ diff -u Barrage_generated.h ../../deephaven-core/cpp-client/deephaven/client/flatbuf/deephaven/flatbuf/Barrage_generated.h  | wc -l
0
```

Reply:
Ah, got it. I hadn't realized I need to ask git to fetch tags explicitly with `git fetch --tags`. Thanks